.gitignore

```diff
@@ -42,5 +42,8 @@ coverage.xml
 .vscode
 .idea
 
+#Poetry
+poetry.lock
```

Comment:
It's recommended to keep this in VC: https://python-poetry.org/docs/basic-usage/#committing-your-poetrylock-file-to-version-control
Unless we feel this is causing a specific issue for us?